### ISSUE TITLE
Can't insert a AgileReview comment in code comment

### ISSUE THREAD

**Issue by ghost** [AgileReview-Project/AgileReview-EclipsePlugin#5]
Is it possible to add an Agile Review comment just in a line in a code comment? I didn't mark the lines 3-8 it's just a code comment and the plugin inserts two Agile Review comments (c2), see picture.
![commentwarning](https://user-images.githubusercontent.com/21191939/36194146-eea4eeb0-1168-11e8-846a-70fea22eba19.JPG)

Thank you, regards

**Comment by maybeec** [comment]
No, you cannot comment a single line in a multi-line comment due to the fact, that the AgileReview comment hooks are also introduced by the same syntax, which would break multi-line comments.

The "two comments" added by agilereview are just the starting and ending tag for the AgileReview comment. They automatically wrap the whole comment as of the previous described issue. 

Introducing the comment hooks in the code, we enabled refactoring safety. You can move the file, write new code into the file without loosing track of the AgileReview comments.

**Comment by biancode** [comment]
Hi Malte,

a suggestion on that topic - shouldn't the end of the tag be a group of characters to parse exactly?
like:  **/*-?**|SAPFin|lands|c1|**?-*/**

In you up to date release we have **?*/** for end tag, which also could come from a question in comments like /*How are you?*/

What do you think?

cya
Klaus

**Comment by maybeec** [comment]
No, the tags have to have the complete structure including the pipes separating the metadata to be interpreted by AgileReview. It is most uncommon, that you will have such a structure. 
So your example of a question should not be handled by AgileReview in any case.

**Comment by biancode** [comment]
hm, but it looks like a wrong parsing for your tags, because of the smashing if I add two review comments in one file, which has regular comments inside

![grafik](https://user-images.githubusercontent.com/4369867/36838265-087837f2-1d3f-11e8-9d74-79e41f139dca.png)

the above comment was added in line 79, but goes up where it is now

**Comment by maybeec** [comment]
indeed, this warning and the upcoming adaption is nonsense and could be prevented. Can you raise an issue in https://github.com/AgileReview-Project/AgileReview-Legacy-Plugin as well? Thanks!

**Comment by biancode** [comment]
I've added the line 85 review before and than I added the line 79 and boom - all is marked and that is wrong

**Comment by maybeec** [comment]
please describe the steps to reproduce and the failure and intended behavior in the new issue
